### PR TITLE
Update the user support page

### PR DESCRIPTION
**Pull request by louzoid-gds**
Brought the content more up to date and rearranged it slightly

### CODE REVIEW

File: source/standards/user-support.html.md.erb [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
 ### 1st line user support
 
 1st line support provide resolution for basic user enquiries and triage frequent, repeatable incidents. For example, issues signing in.
 
+The GDS User Support team provides 1st line support for many teams in GDS. Contact the User Support team using the [#user-support Slack channel].
+
 ### 2nd line technical support
 
-2nd line support provide direct incident resolution and triage user requests. For example, when users receive timeout errors or performance issues.
+2nd line support provide direct incident resolution (see [incident management]((incident-management.html))) and triage user requests. For example, when users receive timeout errors or performance issues.
```
Comment (timblair) [inline source/standards/user-support.html.md.erb:25]:
```suggestion
2nd line support provide direct incident resolution (see [incident management](/incident-management.html)) and triage user requests. For example, when users receive timeout errors or performance issues.
```

File: source/standards/user-support.html.md.erb [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
-Support Operations require GDS teams to use [Zendesk](https://www.zendesk.com/) to manage communications with service users. Support Operations work with GDS teams to help manage Zendesk setup and configuration, for example with:
-
-- choosing agent types, groups and ‘pass-through’ to other government departments
-- [General Data Protection Regulation (GDPR)](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/) considerations, such as data disclosure and data retention
+GDS teams operate support models that are aligned to the needs of the users that they are supporting. For example, the [GOV.UK Pay](https://www.payments.service.gov.uk/) team supports users working in government departments that are using GOV.UK Pay or considering doing so. [GOV.UK](https://www.gov.uk/) and [One Login](https://www.sign-in.service.gov.uk/) support both government and non-government users.
 
 ## Support lines for your service
 
-GDS teams provide support in 3 groups, known as ‘lines’, for each product.
+However the support model varies, teams will have multiple 'lines' of support
```
Comment (timblair) [inline source/standards/user-support.html.md.erb:15]:
```suggestion
However the support model varies, teams will have multiple 'lines' of support.
```

**Review by timblair — CHANGES_REQUESTED** [review]
LGTM, other than fixing this link and adding a full stop.

**Comment by louzoid-gds** [comment]
Thanks Tim! Once the user support folks have got back to me, I'll move this PR out of draft.

**Review by timblair — APPROVED** [review]

**Pull request merged**